docker/DOCKERFILE

```diff
@@ -0,0 +1,11 @@
+LABEL maintainer=omelisra123@gmail.com
+#https://hub.docker.com/r/patwoz/expo-cli
+#baseline of this DOCKERFILE come from this repo
+FROM patwoz/expo-cli:3 
+
+RUN mkdir/CAMA
```

Comment:
docker build runs on that line into the following error:

> The command '/bin/sh -c mkdir/CAMA' returned a non-zero code: 127

System: Win10 (Home)
Docker: Docker Toolbox